Svc/DpCatalog/test/ut/DpCatalogTester.cpp

```diff
@@ -622,7 +626,8 @@ namespace Svc {
 
                 // fill the input entries with random priorities
                 for (FwIndexType entry = 0; entry < static_cast<FwIndexType>(FW_NUM_ARRAY_ELEMENTS(inputs)); entry++) {
-                    U32 randVal = rand()%NUM_ENTRIES;
+                    std::uniform_int_distribution<U32> numEntriesDis(0, NUM_ENTRIES - 1);
```

Comment:
Just FYI in case you haven't seen it yet - for randomness in test code we have the `STest` framework that has a couple easy-to-use methods.

https://github.com/nasa/fprime/blob/1fe6a7cb046c2f443e57a685edac22281b4523f1/STest/STest/Random/Random.hpp#L70-L77